.gitignore

```diff
@@ -1,2 +1,4 @@
 /bin/
 */coverage.txt
+*.exe
```

Comment:
Ultimately yes - there needs to be some work on the native build process for Windows to do that (outside of make). But this is consistent with docker/docker at the moment.